File: 백준/Bronze/10818. 최소， 최대/최소， 최대.cc
#include <iostream>
using namespace std;


int main()
{
	int n;
	cin >> n;

	int max = -1000000, min = 1000000;

	for (int i = 0; i < n; i++) {
		int val;
		cin >> val;

		if (val > max) max = val;
		if (val < min) min = val;
	}

	cout << min << " " << max;

	return 0;
}

File: 백준/Bronze/10818. 최소， 최대/최소， 최대.java
import java.util.Scanner;

class Main {
	public static void main(String[] args) throws Exception {
		Scanner scanner = new Scanner(System.in);
		int n = scanner.nextInt();
		int [] arr = new int[n];
		int minAns = 1000000;
		int maxAns = -1000000;
		for(int i = 0; i<n; i++){
			arr[i] = scanner.nextInt();
			minAns = Math.min(minAns,arr[i]);
			maxAns = Math.max(maxAns,arr[i]);
		}
		System.out.print(minAns + " " + maxAns);
	}
}